pkg/model/metric.go

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+This module implements the common data structure.
+*/
+
+package model
+
+
+type MetricSpec struct {
+	// Following are the labels associated with Metric, same as Prometheus labels
+
+
+
+	//Example: {device="dm-0",instance="121.244.95.60:12419",job="prometheus"}
+
+
+
+	// Instance ID -\> volumeID/NodeID
+
+	InstanceID string `json:"InstanceID,omitempty"`
+
+
+
+	// instance name -\> volume name / node name etc.
+
+	InstanceName string `json:"InstanceName,omitempty"`
```

Comment:
Please change the first letter of json name to lower case, for example `instanceName`.

Reply:
done